lib/game.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+class Game
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @player1_played = []
+    @player2_played = []
+    @board_array = []
+    @box_played = []
+    @current_player = 1
+    @winning_combos = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]]
+    @ct = 0
+  end
+
+  def board
+    # >>> This creates the board and marks every grid with a number 1 - 9
+    # for i in 1..9
+    #   @board_array.push(" [ #{i} ] ").each_slice(3)
+    # end
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

Reply:
Thank you! We forgot to set our development branch as the default and had made our PR to our master with this overlook.